deepchecks/vision/metrics_utils/metrics.py

```diff
@@ -129,7 +130,10 @@ def validate_prediction(batch_predictions: t.Any, dataset: VisionDataset, eps: f
         )
 
 
-def calculate_metrics(metrics: t.List[Metric], dataset: VisionDataset, model: nn.Module,
+def calculate_metrics(metrics: t.List[Metric],
+                      dataset: VisionDataset,
+                      data_loader: DataLoader,
```

Comment:
why have both dataset and it's dataloader? I get it that it's more convenient for the robustness check where the dataloader is modified, but don't think it's reason enough.

Reply:
A. we might have 2 data loaders in VIsionDataset (normal one and sample one)
B. Feel free to share another solution, I couldn't find